### PR TITLE
Feature/63

### LINKED ISSUE #63: Add example demo links to nav
As a developer, I want a "jump list" of example links to existing pages so that I can demo the application to others without having to memorize urls to individual pages.

We haven't figured out how the navigation should work in the app yet, but I think it would be useful to populate the nav with some demo links.

cc @jnmarcus

### PR DESCRIPTION
**Pull request by adborden**
Fixes #63 

- Created a temporary demo nav with links to existing pages so you don't have to memorize URLs.
- Added tests for the nav.
- Created a more dynamic nav, rather than hard-coding the items in the template, it's now in the controller. Ideally, this would be passed into the directive where used.
- Renamed the controller, which was uppercase, to camelcase.

screenshot from 2016-01-03 15-16-45
Created a demo nav.

### CODE REVIEW

File: app/components/common/appMainNav/appMainNavController.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  $scope.mainNavMenu = [
+    {sref: 'home', title: 'Home'}
+  ];
+
+  $scope.demoNavMenu = [
+    {sref: 'appMain.examplePage1', title: 'Example page 1'},
+    {sref: 'appMain.city.money({fips_id: 6075})', title: 'Example city'},
+    {sref: 'appMain.committee.main({committee_id: 1})', title: 'Example committee'},
+    {sref: 'appMain.measure.index({measure_id: 1})', title: 'Example ballot measure'}
```
Comment (tdooner) [inline app/components/common/appMainNav/appMainNavController.js]: :cool:

**Comment by tdooner** [comment]
:+1:

**Comment by bcipolli** [comment]
I don't think PRs should sit for more than 2-3 days on this project, at this phase. Particularly ones that have been reviewed by even one person. 

We don't have to be perfect at this stage, and there are few enough devs that this affects very few people. Let's be agile :)

**Comment by tdooner** [comment]
@bcipolli I agree. The cost of breaking things is low.

**Pull request merged**